README.md

```diff
@@ -62,6 +62,140 @@ settings = OpenActive::DatasetSite::Settings.new(
 )
 ```
 
+
+### Feed-level customisation
+If you need to do feed specific overrides, then you may do this by overriding the method.
+```ruby
+settings = Class.new(OpenActive::DatasetSite::Settings) do
+  def data_download(feed_type)
+    val = super(feed_type)
+
+    case feed_type
+    when OpenActive::DatasetSite::FeedType::SESSION_SERIES
+      val.content_url = open_data_feed_base_url + "session_series"
+    end
+
+    val
+  end
+end.new(
+  data_feed_types: [
+    OpenActive::DatasetSite::FeedType::FACILITY_USE,
+    OpenActive::DatasetSite::FeedType::SCHEDULED_SESSION,
+    OpenActive::DatasetSite::FeedType::SESSION_SERIES,
+    OpenActive::DatasetSite::FeedType::SLOT,
+  ],
+  # rest of your settings here.
+)
+```
+
+To match the PHP/.NET usage, you may alternatively use this approach, however it's less advised as there are more details to get right (and it risks a mismatch between the downloads and the advertised feed types):
+```ruby
+settings = OpenActive::DatasetSite::Settings.new(
+    # your settings here,
+    data_feed_types: [
+      OpenActive::DatasetSite::FeedType::SESSION_SERIES,
+    ],
+    data_downloads: [
+        OpenActive::Models::DataDownload.new(
+          name: "SessionSeries",
+          additional_type: "https://openactive.io/SessionSeries",
+          encoding_format:  OpenActive::DatasetSite::Meta.RPDE_MEDIA_TYPE,
+          content_url: open_data_feed_base_url + "session-series",
+        )
+    ]
+)
+```
+
+### Dataset
+```ruby
+dataset = OpenActive::Models::Dataset.new(
+  id: "http://example.com/dataset/",
+  description:
+    "Near real-time availability and rich descriptions relating to the facilities and sessions available from Simpleweb",
+  url: "http://example.com/dataset/",
+  dateModified: "2019-12-09T15:36:15+00:00",
+  keywords:
+    ["Facilities",
+     "Sessions",
+     "Activities",
+     "Sports",
+     "Physical Activity",
+     "OpenActive"],
+  schemaVersion: "https://www.openactive.io/modelling-opportunity-data/2.0/",
```

Comment:
Would probably either change the casing to snake_case or add a note about internal normalization of casing done by the package, just for consistency/clarity.